website/docs/en/guide/start/index.mdx

```diff
@@ -1,12 +1,30 @@
 # Introduction
 
+Rslib is a library build tool that leverages the excellent configurations and plugins of Rsbuild, empowering library developers to take advantage of the extensive knowledge and ecosystem of webpack and Rspack.
```

Comment:
Should we align the basic introduction with the content of README.md?

https://github.com/web-infra-dev/rslib?tab=readme-ov-file#rslib

Reply:
updated readme.